app/containers/ItunesProvider/ItunesDetailContainer/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { memo } from 'react';
+import { selectSongs } from '../selectors';
+import { Collapse } from 'antd';
+import { createStructuredSelector } from 'reselect';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { injectIntl } from 'react-intl';
+import { injectSaga } from 'redux-injectors';
+import styled from 'styled-components';
+import ItunesProviderSaga from '../saga';
+import PropTypes from 'prop-types';
+import { T } from '@app/components/T/index';
+
+const { Panel } = Collapse;
+
+const Wrapper = styled.div`
+  margin: auto;
+  padding: 20px 0px 20px;
+  max-width: 500px;
+`;
+
+const ItunesDetailContainer = ({ match, songs }) => {
+  const getSong = (id) => {
+    const foundSong = songs.results.find((song) => song.trackId === parseInt(id));
+
+    if (foundSong) {
+      const details = Object.entries(foundSong).map((entry, idx) => {
+        return (
+          <Panel header={entry[0]} key={idx}>
```

Comment:
this is too simplistic for the detail container. Add more details and use more components from antd